src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,34 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected Character taskType;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.taskType = 'T';
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
You might want to add comments on a different line to make the code more readable